components/org.wso2.carbon.micro.integrator.extensions/org.wso2.carbon.micro.integrator.management.apis/src/main/java/org/wso2/carbon/micro/integrator/management/apis/Constants.java

```diff
@@ -30,6 +30,8 @@
     public static final String PREFIX_SEQUENCES = "/sequences";
     public static final String PREFIX_DATA_SERVICES = "/data-services";
     public static final String PREFIX_TEMPLATES = "/templates";
+    public static final String PREFIX_MESSAGE_STORE = "/message-stores";
+
```

Comment:
Shall we format code by removing this extra line?

Reply:
line has been kept to separate prefix constants from others